third_party/chromium/media/filters/source_buffer_stream.cc

```diff
@@ -199,13 +205,21 @@ SourceBufferStream::SourceBufferStream(const VideoDecoderConfig& video_config,
       highest_output_buffer_timestamp_(kNoTimestamp),
       max_interbuffer_distance_(
           base::Milliseconds(kMinimumInterbufferDistanceInMs)),
+#if defined(STARBOARD)
+      memory_limit_(std::max(budget_override,
+          GetDemuxerStreamVideoMemoryLimit(Demuxer::DemuxerTypes::kChunkDemuxer,
+                                           &video_config,
+                                           mime_type_))) {
```

Comment:
Base on the code, it seems that the memory limit can be increased, but not reduced.  This is ok (though not ideal), but let's state that in the commit message, and put a comment in `media_module.cc` where the JS facing setting is set.